### PR TITLE
Annotate haumea-rings.png as CC0-1.0

### PR DESCRIPTION
**Pull request by ajtribick**
* Per discussion on Celestia discord "I assign it to be public domain" (@Askaniy)
* Per REUSE documentation, use the CC0 license for this case

### CODE REVIEW

File: textures/lores/haumea-rings.png.license [hunk 1]
```diff
@@ -0,0 +1,3 @@
+SPDX-FileCopyrightText: 2019 Askaniy
```
Comment (Askaniy) [inline textures/lores/haumea-rings.png.license:1]: 2018 actually

Comment (ajtribick) [reply textures/lores/haumea-rings.png.license]: thanks, updated

Comment (375gnu) [reply textures/lores/haumea-rings.png.license]: should we use full names where possible?

Comment (ajtribick) [reply textures/lores/haumea-rings.png.license]: updated

**Review by 375gnu — APPROVED** [review]

**Pull request merged**